packages/react/src/auto/mui/index.ts

```diff
@@ -1,22 +1,23 @@
-export * from "./MUIAutoForm.js";
-export { MUIAutoForm as AutoForm } from "./MUIAutoForm.js";
-export { MUIAutoBooleanInput as AutoBooleanInput } from "./inputs/MUIAutoBooleanInput.js";
-export { MUIAutoDateTimePicker as DateTimePicker } from "./inputs/MUIAutoDateTimePicker.js";
-export { MUIAutoEnumInput as AutoEnumInput } from "./inputs/MUIAutoEnumInput.js";
-export { MUIAutoFileInput as AutoFileInput } from "./inputs/MUIAutoFileInput.js";
-export { MUIAutoHiddenInput as AutoHiddenInput } from "./inputs/MUIAutoHiddenInput.js";
-export { MUIAutoInput as AutoInput } from "./inputs/MUIAutoInput.js";
-export { MUIAutoJSONInput as AutoJSONInput } from "./inputs/MUIAutoJSONInput.js";
-export { MUIAutoRolesInput as AutoRolesInput } from "./inputs/MUIAutoRolesInput.js";
-export {
-  MUIAutoTextInput as AutoColorInput,
-  MUIAutoTextInput as AutoEmailInput,
-  MUIAutoTextInput as AutoNumberInput,
-  MUIAutoTextInput as AutoStringInput,
-  MUIAutoTextInput as AutoTextInput,
-  MUIAutoTextInput as AutoUrlInput,
-} from "./inputs/MUIAutoTextInput.js";
-export { MUIAutoBelongsToInput as AutoBelongsToInput } from "./inputs/relationships/MUIAutoBelongsToInput.js";
-export { MUIAutoHasManyInput as AutoHasManyInput } from "./inputs/relationships/MUIAutoHasManyInput.js";
-export { MUIAutoSubmit as AutoSubmit } from "./submit/MUIAutoSubmit.js";
-export { MUISubmitResultBanner as SubmitResultBanner } from "./submit/MUISubmitResultBanner.js";
+// The MUI auto components have been temporarily removed. See GGT-6770.
```

Comment:
I don't think we need to include the ticket number here. That is only accessible to Gadget people